.eslintignore

```diff
@@ -2,5 +2,7 @@ node_modules/
 coverage/
 config/
 lib/
+es/
+umd/
```

Comment:
Should we include tmp here as well just in case?
```suggestion
umd/
tmp/
```

Reply:
added